frontend/src/styles/about_us/ValueCard.module.css

```diff
@@ -42,58 +39,66 @@
   .valueCard {
     width: 357px;
     height: 286px;
-    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackground1.svg);
-    background-size: cover;
-    transition: background-image 300ms ease-in;
-  }
-
-  .valueCard:hover {
-    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackgroundHover1.svg);
+    background: None;
   }
 
   .valueCard.rev {
-    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackground2.svg);
+    background: None;
   }
 
-  .valueCard.rev:hover {
-    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackgroundHover2.svg);
-  }
-
-  .valueCardTextMain {
-    position: absolute;
+  .valueCard h2 {
     font-size: 34px;
     line-height: 39px;
-    width: 181px;
-    height: 80px;
+    font-weight: 700px;
     text-align: center;
-
-    opacity: 1;
-    transition: opacity 300ms ease-in;
+    width: 181px;
   }
 
   .valueCardTextSub {
-    position: absolute;
-    width: 234px;
-    height: 147px;
     text-align: left;
+  }
+
+  .cardContentWrapper {
+    height: inherit;
+    width: inherit;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .cardFront {
+    position: absolute;
+    height: inherit;
+    width: inherit;
+    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackground1.svg);
 
-    opacity: 0;
     transition: opacity 300ms ease-in;
   }
 
-  .valueCard:hover .valueCardBg1 {
+  .rev .cardFront {
+    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackground2.svg);
+  }
+
+  .cardBack {
+    position: absolute;
+    height: inherit;
+    width: inherit;
+    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackgroundHover1.svg);
     opacity: 0;
+
+    transition: opacity 300ms ease-in;
   }
 
-  .valueCard:hover .valueCardBg2 {
-    opacity: 1;
+  .rev .cardBack {
+    background-image: url(../../components/assets/ValueCardBackgrounds/ValueCardBackgroundHover2.svg);
   }
 
-  .valueCard:hover .valueCardTextMain {
+  .valueCard:hover .cardFront {
     opacity: 0;
   }
 
-  .valueCard:hover .valueCardTextSub {
+  .valueCard:hover .cardBack {
```

Comment:
im ngl i didnt hard read this diff because its huge but im signing off bc ik you write good code TTuTT

Reply:
it looks good though nothing sus